packages/rag/reranker.py

```
@@ -0,0 +1,114 @@
+    documents = [
+        {"id": str(chunk["id"]), "doc": chunk["content"]}
+        for chunk in chunks
+    ]
+
+    data = {
+        "documents": documents,
+        "query": query,
+        "maxTokens": max_tokens,
+    }
+
+    response = requests.post(url, headers=headers, json=data)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)